example/README.md

```diff
@@ -26,7 +26,8 @@ A Provision Watcher contains the following fields:
 `Profile`: The name of a DeviceProfile which should be assigned to new devices which meet the given criteria  
 `AdminState`: The initial Administrative State for new devices which meet the given criteria  
  
-A candidate new device passes a ProvisionWatcher if all of the Identifiers match, and none of the BlockingIdentifiers.
+A candidate new device passes a ProvisionWatcher if all of the Intifiers match, and none of the BlockingIdentifiers.
```

Comment:
change `Intifiers` to `Identifiers` and `BlockingIdentifiers` to `Blocking Identifiers match`